src/main/webapp/content/scss/themes/_default-variables.scss

```diff
@@ -504,6 +506,11 @@ $apollon-gray: $gray-200;
 $apollon-gray-variant: $gray-700;
 $apollon-grid: rgba(36, 39, 36, 0.1);
 
+// Feedback Suggestions (Athena)
+$feedback-suggestions-primary: rgb(181, 78, 254, 1);
+$feedback-suggestions-background: lighten($feedback-suggestions-primary, 24%);
```

Comment:
Is there a reason, why $feedback-suggestions-background is darkened in dark-mode by 50% but lightened tin light-mode by just 24%?

Reply:
Only because it looked better :) I tried using the same percentage, but the saturation of one of the two colors was never right.

Reply:
For reference, this is how the colors currently look:
<img width="670" alt="screenshot-2023-09-15_001977" src="https://github.com/ls1intum/Artemis/assets/9006596/6b9ab14d-5b63-4030-84be-3cf05486b5ea">
<img width="661" alt="screenshot-2023-09-15_001978" src="https://github.com/ls1intum/Artemis/assets/9006596/01347d2d-1435-4b49-b98b-9c86f1eaf074">